activiti-cloud-runtime-bundle-service/activiti-cloud-services-runtime-bundle/activiti-cloud-services-connectors/src/main/java/org/activiti/services/connectors/behavior/MQServiceTaskBehavior.java

```diff
@@ -60,10 +60,10 @@ public MQServiceTaskBehavior(
     }
 
     @Override
-    public void execute(DelegateExecution execution) {
+    public DelegateExecutionOutcome apply(DelegateExecution execution) {
         if (defaultServiceTaskBehavior.hasConnectorBean(execution)) {
             // use de default implementation -> directly call a bean
-            defaultServiceTaskBehavior.execute(execution);
+            return defaultServiceTaskBehavior.apply(execution);
         } else {
```

Comment:
Now with the addition of the return in the if, can we keep this code out of the else?